.eslintrc.json

```diff
@@ -0,0 +1,19 @@
+{
+  "extends": ["bloq", "prettier"],
+  "overrides": [
+    {
+      "extends": ["plugin:@typescript-eslint/recommended", "prettier"],
+      "files": ["*.ts"],
+      "parser": "@typescript-eslint/parser",
+      "plugins": ["@typescript-eslint"]
+    },
```

Comment:
Can we update `eslint-config-bloq` and use `bloq/typescript` instead?

Reply:
Done!